### PR TITLE
docs(style): make more responsive for mobile

### PR DESCRIPTION
**Pull request by mistercrunch**
see title ^^^
Screen Shot 2020-09-14 at 9 42 55 AM
Screen Shot 2020-09-14 at 9 43 00 AM

Screen Shot 2020-09-14 at 9 43 09 AM
Screen Shot 2020-09-14 at 9 43 14 AM

### CODE REVIEW

**Review by pkdotson — APPROVED** [review]
LGTG!

File: docs/src/pages/resources.tsx [hunk 1]
```diff
@@ -228,17 +186,18 @@ const Resources = () => {
             </Modal>
             <h2>Videos</h2>
             <Card>
-              {youtubeIds.map(([idx, ids, cardTitle]) => (
+              {youtubeIds.map(([idx, ids, title]) => (
```
Comment (rusackas) [inline docs/src/pages/resources.tsx:189]: @mistercrunch @pkdotson NIT that shouldn't hold up this PR: We can remove the index from the data arrays, and let the `map` provide it intrinsically, e.g. `{youtubeIds.map(([ids, title], idx) => (`

File: docs/src/pages/resources.tsx [hunk 2]
```diff
@@ -130,28 +99,40 @@ const resourcesContainer = css`
...
-      }
-    }
   }
 `;
 
+interface featureProps {
+  title: string,
+  descr: Array,
+}
+const LinkSection = ({ title, links }: featureProps) => (
```
Comment (rusackas) [inline docs/src/pages/resources.tsx:109]: ❤️ DRY

File: docs/src/pages/resources.tsx [hunk 3]
```diff
@@ -52,76 +54,43 @@ const installationLinks = [
...
-  "https://www.youtube.com/embed/z350Gbi463I"
+  'https://www.youtube.com/embed/24XDOkGJrEY',
+  'https://www.youtube.com/embed/AqousXQ7YHw',
+  'https://www.youtube.com/embed/JGeIHrQYhIs',
+  'https://www.youtube.com/embed/z350Gbi463I',
 ];
 
 const youtubeIds = [
   [
     0,
```
Comment (rusackas) [inline docs/src/pages/resources.tsx:66]: These are the indices we can skip, as per the comment further down.

Comment (rusackas) [reply docs/src/pages/resources.tsx:66]: I don't think this array is necessary. The youtubeIds data structure below could include or reconstruct these strings.

File: docs/src/pages/community.tsx [hunk 4]
```diff
@@ -125,7 +123,11 @@ const Community = () => {
               dataSource={links}
               renderItem={([href, link, post]) => (
                 <List.Item>
-                  <a href={href}>{link}</a> - {post}
+                  <a href={href}>{link}</a>
+                  {' '}
```
Comment (rusackas) [inline docs/src/pages/community.tsx:127]: To me, these are weird... call me old school, but I'd almost rather see an `&nbsp;` ¯\\\_(ツ)_/¯

File: docs/README.md [hunk 5]
```diff
@@ -31,3 +31,36 @@ npm install
 npm run start
 # navigate to localhost:8000`
 ```
+
+## To Publish
```
Comment (rusackas) [inline docs/README.md:35]: ❤️ docs in our docs

**Review by rusackas — APPROVED** [review]
Noted a few nits, namely one around a data structure cleanup opportunity, but otherwise LGTM!

File: docs/src/pages/community.tsx (hunk 4 above)
Comment (mistercrunch) [reply docs/src/pages/community.tsx:127]: I think the linter might have done this(?), I have to dbl check

**Comment by mistercrunch** [comment]
Good input, merging as-is and leaving some of this for a later PR.

**Pull request merged**